### DISCUSSION TITLE
Is there way of Viewing?

### DISCUSSION THREAD

**Discussion by polyefko** [flybywiresim/aircraft#2956] [Help] [answered]
I hope I can explain what Im asking.Lol,  When I look at the outside view of the aircraft (say while starting the taxi movement) and I turn to head out to the runway..my view also moves with the turn of the aircraft. What I want is to view the aircraft from the front and when I turn to head towards the runway I want my view not moving with the turning of the aircraft. Thats how I view my plane in XPLANE 11....my taxi view just stays as the plane moves left or right.

**Comment by Inaxair** [comment, marked as answer]
Hello

This is a simulator feature which we don't control unfortunately. However, there might have mods in flightsim.to or other stuff. You can also ask Microsoft Flight Simulator team here: https://forums.flightsimulator.com/

Thanks!
Inaxair from FlybyWire Simulation

**Comment by hotshotp** [comment]
turn off smart camera

**Reply by robinottawa** [reply]
Isn't that what smart camera does? I was planning to try it. Please let me know if that works or not. I use the same view you are talking about and just have my mouse ready to adjust the view so I can see where I'm taxiing while I turn corners, etc.

**Reply by hotshotp** [reply]
smart camera will snap to things, im not sure what you are saying but try toggling it on and off, to adjust camera freely turn off the smart cam
